src/DssSpell.t.sol

```diff
@@ -436,25 +429,6 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(SpellActionLike(spell.action()).dao_resolutions(), comma_separated_resolutions, "dao_resolutions/invalid-format");
     }
 
-    // SPARK TESTS
-    function testSparkSpellIsExecuted() private { // make private to disable
-        address SPARK_PROXY = 0x4e847915D8a9f2Ab0cDf2FC2FD0A30428F25665d;
-        address SPARK_SPELL = address(0);
-
-        vm.expectCall(
-            SPARK_PROXY,
-            /* value = */ 0,
-            abi.encodeCall(
-                ProxyLike(SPARK_PROXY).exec,
-                (SPARK_SPELL, abi.encodeWithSignature("execute()"))
-            )
-        );
-
-        _vote(address(spell));
-        _scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-    }
-
```

Comment:
Spark is not maintained in Goerli anymore. It makes no sense to keep this test here.